nototools/unicode_data.py

```diff
@@ -885,6 +1017,11 @@ def _load_emoji_group_data():
   with open_unicode_data_file('emoji-test.txt') as f:
     text = f.read()
   group_list = _read_emoji_test_data(text)
+
+  # patch with android items
+  patch = _get_android_order_patch()
+  group_list = _apply_order_patch(patch, group_list)
```

Comment:
just a thought: 'order' might as well be dropped from both function names above.